examples/hello-world/hello-cross-val/jobs/hello-cross-val/app/custom/train.py

```diff
@@ -0,0 +1,201 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
If this is new file may need to be 2024?

Reply:
updated